api/audit/constants.py

```diff
@@ -0,0 +1,32 @@
+FEATURE_CREATED_MESSAGE = "New Flag / Remote Config created: %s"
+FEATURE_DELETED_MESSAGE = "Flag / Remote Config Deleted: %s"
+FEATURE_UPDATED_MESSAGE = "Flag / Remote Config updated: %s"
+SEGMENT_CREATED_MESSAGE = "New Segment created: %s"
+SEGMENT_UPDATED_MESSAGE = "Segment updated: %s"
+FEATURE_SEGMENT_UPDATED_MESSAGE = (
+    "Segment rules updated for flag: %s in environment: %s"
+)
+ENVIRONMENT_CREATED_MESSAGE = "New Environment created: %s"
+ENVIRONMENT_UPDATED_MESSAGE = "Environment updated: %s"
+FEATURE_STATE_UPDATED_MESSAGE = (
+    "Flag state / Remote Config value updated for feature: %s"
+)
+FEATURE_STATE_WENT_LIVE_MESSAGE = (
+    "Scheduled change to Flag state / Remote config value went live for feature: %s by"
+    " Change Request: %s"
+)
+IDENTITY_FEATURE_STATE_UPDATED_MESSAGE = (
+    "Flag state / Remote config value updated for feature '%s' and identity '%s'"
+)
+SEGMENT_FEATURE_STATE_UPDATED_MESSAGE = (
+    "Flag state / Remote config value updated for feature '%s' and segment '%s'"
+)
+IDENTITY_FEATURE_STATE_DELETED_MESSAGE = (
```

Comment:
Do we want to capture the changed value here as well? I think some people have asked for it, right?